features/install.feature

```diff
@@ -5,7 +5,45 @@ Feature: Installation
   So that I can start testing without repetitive setup work.
 
 
-  Scenario: fresh install
+  Scenario: install with a .js test helper
+    Given I am working on a Node.js project
+    When I run "mycha install" and enter "spec/spec_helper.js"
```

Comment:
On asf we did specs inline with source, and the spec folder was just for helpers.